idna/core.py

```diff
@@ -259,14 +261,14 @@ def check_label(label):
     check_bidi(label)
 
 
-def alabel(label):
+def alabel(label: str) -> bytes:
 
     try:
-        label = label.encode('ascii')
-        ulabel(label)
-        if not valid_label_length(label):
+        label_bytes = label.encode('ascii')
```

Comment:
Had to make this `label_bytes` change because mypy doesn't like setting `bytes` to a variable previously typed as `str`. Used `label_bytes` throughout in this file.